### PR TITLE
Update for QS/MP

### PR DESCRIPTION
**Pull request by alexhung**
- Update RT and Xray versions to latest
- Remove `KeystorePassword` and `AnsibleVaultPass` parameters, and use db password instead
- Pass in RT license secret name to instance user data script, fetch whole secret using aws cli, and use Ansible to parse out the license text.
- Update binarystore configuration so both Pro and HA have same configuration.
- Update RT and Xray AMI creation templates
  - Simplify AMIs mappings
  - Add python 3 updates for instance user data
  - Add volume attaching
- Rename 'ArtifactoryMaster' to 'ArtifactoryPrimary'

### CODE REVIEW

File: templates/jfrog-ami-creation.template.yaml [hunk 1]
```diff
@@ -203,25 +199,27 @@ Resources:
...
-              content: !Sub |
+              content: !Sub
+                - |
                   # Base install for JFrogAMIInstance
                   - import_playbook: artifactory-ami.yml
                     vars:
                       ami_creation: true
                       artifactory_version: ${ArtifactoryVersion}
-                      artifactory_ha_enabled: true
+                      artifactory_ha_enabled: false
```
Comment (alexhung) [inline templates/jfrog-ami-creation.template.yaml:209]: @vinayagg Should this be `true` or `false` for QS/MP? I now think it should be 'true' but not sure

Comment (vinayagg) [reply templates/jfrog-ami-creation.template.yaml]: true

**Review by vinayagg — APPROVED** [review]

**Comment by alexhung** [comment]
@vinayagg I think I updated all the versions.

**Comment by vinayagg** [comment]
Lets update these files too
https://github.com/jfrog/quickstart-jfrog-artifactory/blob/update-ami-creations/templates/jfrog-artifactory-ec2-master.template.yaml

there should be 4 files - master, existing-vpc, pro-master, pro-existing vpc

**Review by vinayagg — APPROVED** [review]

**Pull request merged**